src/main/java/io/quarkus/registry/app/DatabaseRegistryClient.java

```diff
@@ -217,6 +281,16 @@ private io.quarkus.registry.catalog.PlatformRelease.Mutable toClientPlatformRele
                 .setQuarkusCoreVersion(platformRelease.quarkusCoreVersion);
     }
 
+    private String getNonPlatformCoords(String quarkusVersion) {
```

Comment:
It would be better if this is moved to the `MavenConfig` class

Reply:
Done! I agree, that's nicer. 
With that change, nothing was using the original `getNonPlatformExtensionCoords`, so I eliminated it. I also decided that `getNonPlatformCoords` probably should be called `getNonPlatformExtensionCoords`, so I updated the name. 

Let me know if you think mavenConfig.getNonPlatformExtensionCoords was used externally, and I will reinstate the original method as well as the new one.